client/app/collections/application.coffee

```diff
@@ -17,177 +20,10 @@ module.exports = class ApplicationCollection extends BaseCollection
-                description:"kresus description"
-           ,
-                icon:"img/apps/nirc.svg"
-                name:"nirc"
-                displayName:"nIRC"
-                slug:"nirc"
-                git:"https://github.com/frankrousseau/cozy-nirc.git"
-                comment:"community contribution"
-                description:"nirc description"
-           ,
-                icon:"img/apps/notes.svg"
-                name:"notes"
-                displayName:"Note"
-                slug:"notes"
-                git:"https://github.com/cozy/notes.git"
-                comment:"community contribution"
-                description:"notes description"
-           ,
-                icon: "img/apps/owm.svg"
-                name: "owm"
-                displayName:"OWM"
-                slug: "owm"
-                git: "https://github.com/Piour/piour-cozy-owm.git"
-                comment: "community contribution"
-                description: "owm description"
-           ,
-                icon: "img/apps/pfm.svg"
-                name: "mes comptes"
-                displayName:"Mes Comptes"
-                slug: "pfm"
-                git: "https://github.com/seeker89/cozy-pfm.git"
-                comment: "community contribution"
-                description: "pfm description"
-           ,
-                icon:"img/remotestorage-icon.png"
-                name:"remotestorage"
-                displayName:"Remote Storage"
-                slug:"remotestorage"
-                git:"https://github.com/aenario/cozy-remotestorage.git"
-                comment:"community contribution"
-                description:"remote storage description"
-           ,
-                icon:"img/apps/tasky.svg"
-                name:"tasky"
-                displayName:"Tasky"
-                slug:"tasky"
-                git:"https://github.com/jsilvestre/tasky.git"
-                comment:"community contribution"
-                description:"tasky description"
-                color: '#1bda4c'
-           ,
-                icon:"img/apps/todos.svg"
-                name:"todos"
-                displayName:"Todos"
-                slug:"todos"
-                git:"https://github.com/cozy/todos.git"
-                comment:"community contribution"
-                description:"todos description"
-           ,
-                icon:"img/term-icon.png"
-                name:"term"
-                displayName:"Term"
-                slug:"term"
-                git:"https://github.com/alpha14/cozy-term.git"
-                comment:"community contribution"
-                description:"term description"
-        ]
-
-        @reset apps
+    fetchFromMarket: (callback) ->
+        if @apps.length > 0
+            callback @reset(@apps)
+        else
+            client.get 'api/applications/market', (err, body) =>
+                @apps = body.apps
+                callback @reset(@apps)
```

Comment:
since you don't use what is returned by `callback()` when you call the function, I suggest you do:
```
@reset @apps
callback()
```
Because I think it's misleading